### DISCUSSION TITLE
[2021.09.01] 있었는데요 없었습니다

### DISCUSSION THREAD

**Discussion by chaeyeon4u** [pyunji/Hyundai-ITnE-Team_QuadCore#93] [SQL]
[있었는데요 없었습니다](https://programmers.co.kr/learn/courses/30/lessons/59043)

**Comment by pyunji** [comment]
[있었는데요 없었습니다 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/sql/PR_59043.sql)

**Comment by leechang4** [comment]
[있었는데요 없었습니다 solved](https://github.com/leechang4/Hyundai-ITnE-Team_QuadCore/blob/master/lch/SQL/PR_59043.sql)

**Comment by chaeyeon4u** [comment]
[있었는데요 없었습니다 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/SQL/PR_59043.sql)

**Comment by hyk339** [comment]
[있었는데요 없었습니다 solved](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/blob/master/khy/sql/PR_59043.sql)
